utils.js

```diff
@@ -89,6 +95,9 @@ function sendPing(key, window, details) {
   payload[key] = 1;
   Object.assign(payload, details);
 
+  // Send metrics to shield telemetry
+  TCStudy.report(payload);
```

Comment:
We should probably remove/comment out the testpilot code below on the shield branch…